src/renderer/corporate/member/corporate-add-member-container.jsx

```diff
@@ -85,6 +137,37 @@ class CorporateAddMemberContainer extends PureComponent {
 		return identityAttributes.validate(type.content, value, []);
 	}
 
+	validateAttributeType = type => {
+		return ['individual', 'corporate'].includes(type);
+	};
+
+	validateAttributePositions = selectedPositions => {
+		const acceptablePositions = this.props.availablePositions.map(p => p.position);
+		let isError = false;
+		if (!selectedPositions || selectedPositions.size === 0) {
+			return false;
+		}
+		selectedPositions.forEach(p => {
+			isError = isError || !acceptablePositions.includes(p);
+		});
+		return !isError;
+	};
+
+	validateAttributeEquity = shares => {
+		const number = parseInt(shares);
+		return !isNaN(number) && number >= 0 && number <= 100;
+	};
+
+	validateAttributeDid = did => true;
+
+	filterAcceptablePositions = selectedPositions =>
+		selectedPositions.filter(p =>
+			this.props.availablePositions.map(pos => pos.position).includes(p)
+		);
+
+	validateAttributeParentId = parentId =>
+		this.props.companies.find(c => c.identity.id === parentId);
```

Comment:
does it work? I can see an issue here with parentId type, in identity it's integer and in form it might be 'string'

Reply:
It works... it's javascript! 
Fixed on fdfa38c